### PR TITLE
Tizen

### PR DESCRIPTION
**Pull request by davidrogers-unity**
Tizen Emulator support

### CODE REVIEW

File: external/buildscripts/build_runtime_tizen.sh [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
 
 BUILDDIR=/$PWD
 OUTDIR=builds/embedruntimes/tizen
 BUILDSCRIPTSDIR=external/buildscripts
 
-# perl ${BUILDSCRIPTSDIR}/SDKDownloader.pm --repo_name=tizen-sdk --artifacts_folder=artifacts && source ${TIZEN_SDK}/tizen-ndk-env.sh
-perl ${BUILDSCRIPTSDIR}/PrepareTizenNDK.pl -ndk=2.3.0a2 -env=envsetup.sh && source envsetup.sh && source ${TIZEN_NDK_ROOT}/tizen-ndk-env.sh
+#perl ${BUILDSCRIPTSDIR}/PrepareTizenNDK.pl -ndk=2.4.0 -env=envsetup.sh && source envsetup.sh && source ${TIZEN_NDK_ROOT}/tizen-ndk-env.sh
 
-CXXFLAGS="-Os -DHAVE_ARMV6=1 -DARM_FPU_VFP=1 -D__ARM_EABI__ -mno-thumb -march=armv7-a -mfloat-abi=softfp -mfpu=neon -mtune=cortex-a9 \
```
Comment (joncham) [inline external/buildscripts/build_runtime_tizen.sh]: Does our Mono correctly handle thumb instructions? Calling to/from thumb mode?

Comment (davidrogers-unity) [inline external/buildscripts/build_runtime_tizen.sh]: With this we switched from GCC 4.6 to 4.9 on Tizen. It apparently doesn't know the -mno-thumb option anymore.

Comment (davidrogers-unity) [inline external/buildscripts/build_runtime_tizen.sh]: But yeah, everything does still work fine.

Comment (davidrogers-unity) [inline external/buildscripts/build_runtime_tizen.sh]: Any other comments or is this ok to merge?

Comment (joncham) [inline external/buildscripts/build_runtime_tizen.sh]: LGTM

Comment (davidrogers-unity) [inline external/buildscripts/build_runtime_tizen.sh]: Should I do the merge or is that up to someone else?

**Pull request merged**